check_layout.output

```diff
@@ -59,6 +59,9 @@ Layout doesn't define some important keys, missing: loc esc, loc tab
 Duplicate keys: ।
 Layout includes some ASCII punctuation but not all, missing: ", $, ', ^, _, `, |
 2 warnings
+# deva_phonetic_in
+Duplicate keys: ट
```

Comment:
You have `ट` at two places at the same time. Is this intended ?

Reply:
Yes. Since it is a phonetic sound of T, people may expect it to be there but there was another space available so I put it at both places  to make it more accessible.